### PR TITLE
[Windows OS] adding kotlin lang using chocolatey

### PR DESCRIPTION
**Pull request by mikhailshaganov**
# Description
A new tool for Windows OS. Kotlin lang was added using chocolatey software.

#### Related issue:
https://github.com/actions/virtual-environments/issues/3687

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -78,6 +78,12 @@ function Get-GoVersion {
     return "Go $goVersion"
 }
 
+function Get-KotlinVersion {
+    $kotlinVersion = $(kotlinc -version).split(" ")[1]
+    return "Ruby $kotlinVersion"
+}
+
+
```
Comment (AlenaSviridenko) [inline images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:86]: extra line

File: images/win/scripts/Tests/ChocoPackages.Tests.ps1 [hunk 2]
```diff
@@ -96,4 +96,10 @@ Describe "CMake" {
     It "cmake" {
         "cmake --version" | Should -ReturnZeroExitCode
     }
+}
+
+Describe "Kotlinc" {
```
Comment (AlenaSviridenko) [inline images/win/scripts/Tests/ChocoPackages.Tests.ps1:101]: the whole test suite should be named `Kotlin`.

File: images/win/scripts/Tests/ChocoPackages.Tests.ps1 [hunk 3]
```diff
@@ -96,4 +96,10 @@ Describe "CMake" {
...
+    It "kotlinc" {
```
Comment (AlenaSviridenko) [inline images/win/scripts/Tests/ChocoPackages.Tests.ps1:102]: Looks like there should be more tests to match test set for Ubuntu and macOS

**Review by Jbell971 — APPROVED** [review]

File: images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1 [hunk 4]
```diff
@@ -78,6 +78,12 @@ function Get-GoVersion {
     return "Go $goVersion"
 }
 
+function Get-KotlinVersion {
+    $kotlinVersion = $(kotlinc -version).split(" ")[1]
+    return "Kotlin $kotlinVersion"
+}
+
+
```
Comment (AlenaSviridenko) [inline images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1:86]: please, remove extra line

**Review by al-cheb — DISMISSED** [review]

**Comment by AlenaSviridenko** [comment]
Version is empty, please, fix
image

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**